frontend/pages/organizations/[organizationUrl].js

```diff
@@ -313,6 +362,45 @@ async function getProjectsByOrganization(organizationUrl, token, locale) {
   }
 }
 
+async function getChildOrganizationInfo(organizationUrl, token, locale) {
+  const orgs = await getChildOrganizations(organizationUrl);
+  if (orgs != null) {
+    const projects = await getChildProjects(orgs, token, locale);
```

Comment:
This is not a good idea for performance to wait for one API call to finish and then add another. This needs to either be done in one API call or the projects need to be caught afterwards (The first option is better for SEO since Google can then find the projects on the page)

Reply:
The best solution here is probably to alter the endpoint that catches the projects to also return the child projects. Because then pagination would also work as expected (e.g. only the first 10 projects will be shown without scrolling down)